agent/util-scripts/pbench-move-result-tb

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 4; sh-indentation: 4; sh-indent-for-case-alt: + -*-
+
+script_path="$(dirname ${0})"
+script_name="$(basename ${0})"
+pbench_bin="$(realpath -e ${script_path}/..)"
+
+# source the base script
+. "${pbench_bin}"/base
+
+function usage() {
+    printf "usage:\n"
+    printf "${script_name} [--help] <full path to tar ball> <results host> <results path prefix>\n"
+}
+
+# Process options and arguments
+opts=$(getopt -q -o h --longoptions "help" -- "${@}")
+if [[ ${?} -ne 0 ]]; then
+    printf "\n${script_name}: you specified an invalid option\n\n" >&2
+    usage >&2
+    exit 1
+fi
+
+eval set -- "${opts}"
+while true; do
+    case "${1}" in
+	-h|--help)
+	    usage
+	    exit 0
+	    ;;
+	--)
+	    shift;
+	    break;
+	    ;;
+    esac
+done
+
+tarball=$(realpath -e ${1})
+if [[ ! -f "${tarball}" ]]; then
+    error_log "ERROR: tar ball does not exist, ${tarball}"
+    exit 1
+fi
+if [[ ! -f "${tarball}.md5.check" ]]; then
+    error_log "ERROR: tar ball's .md5.check does not exist, ${tarball}.md5.check"
+    exit 1
+fi
+controller_dir=$(dirname ${tarball})
+cnt=$(find ${controller_dir} -type f 2> /dev/null | wc -l 2> /dev/null)
```

Comment:
Any errors in removing a previous result's artifacts will cause cascading errors for all subsequent tarballs: are we OK with that? Maybe we should relax the check to be just for this tarball: `find ....-name "${tarball}.*"` ?

Reply:
Addressed this by checking the error status of the `rm` commands and the `touch` command.